dev/seed.py

```diff
@@ -18,7 +18,7 @@ def get_db_path():
 
 
 def create_random_job(index: int):
-    status = random.choice(["COMPLETED", "FAILED", "IN_PROGRESS", "STOPPED"])
+    status = random.choice(["CREATED", "COMPLETED", "FAILED", "IN_PROGRESS", "STOPPED"])
```

Comment:
Should `QUEUED` also be a possible choice here?

Reply:
Probably, we don't support on OSS side, but I can add it.